cha/implementaion/playback/decoding.tex

```diff
@@ -0,0 +1,48 @@
+\section{Decoding MP3}
+In \cnameref{cha:initial_development} we presented our initial foundation for development, namely the Google sample implementation of a universal media player app.
+In that implementation an Android implementation, \code{android.media.MediaPlayer}, is used to control the playback of audio streams.
+
+The decoding from MP3 to \ac{PCM} is already handled by the base implementation from Google samples, however we have no guarantees when actions occur, as it is left up to this already implemented system and done in correlation with playing the \ac{PCM} data.
+We replace the \code{android.media.MediaPlayer} class with our own implementation in order to obtain more control over the playback and decoding.
```

Comment:
Looking again I just think this needs to go and let the next sentence pick it up. Having it twice in a row is annoying.